sanic/server.py

```diff
@@ -1,47 +1,47 @@
 import asyncio
+from functools import partial
 from inspect import isawaitable
 from signal import SIGINT, SIGTERM
 
-import httptools
+from httptools import HttpRequestParser
+from httptools.parser.errors import HttpParserError
 
 try:
     import uvloop as async_loop
 except ImportError:
     async_loop = asyncio
 
+from .config import Config
 from .log import log
 from .request import Request
 
+CONNECTIONS = set()
```

Comment:
I like using the set, but I think capitals implies it is a constant.  Also we will need to make this a member variable again if we want to support running multiple servers on the same event loop.